LondonBritishLibrary/orient/BLorient3716c.xml

```diff
@@ -0,0 +1,264 @@
+                                </support>
+                                <extent>
+                                    <measure unit="leaf" quantity="1">1</measure>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>2260</height>
+                                        <width>195</width>
+                                    </dimensions>
+                                    <note>Scroll composed of <measure unit="strips" quantity="3">3</measure> strips.</note>
+                                </extent>
+                            </supportDesc>
+                        </objectDesc>
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <desc>Large, rather poor handwriting.</desc>
+                                <date notBefore="1800" notAfter="1867" resp="PRS8999Strelcyn"/>
+                            </handNote>
+                        </handDesc>
+                        <decoDesc>
+                            <decoNote xml:id="d1" type="miniature">
+                                <desc><term key="MagicDrawing">Coloured magic picture</term> without title.</desc>
+                            </decoNote>
+                            <decoNote xml:id="d2" type="miniature">
+                                <desc>Coloured picture representing <persName ref="PRS2548Basilide"/> spearing Satan (indicated with 
+                                    <foreign xml:lang="gez">ቅዱስ፡ ፋሲለደስ፡ ዘከመ፡ ረገዞ፡ ለሰይጣን፡</foreign>) and <persName ref="PRS8065Raguel"/> 
+                                    spearing the Dragon (indicated with <foreign xml:lang="gez">ቅዱስ፡ ራጉኤል፡ ዘከመ፡ ረገዞ፡ ለከይሲ፡</foreign>). Near the 
+                                    Dragon, Satan is depicted holding the souls of the damned (indicated with <foreign xml:lang="gez">ነፍሳት፡</foreign>),
+                                    below <persName ref="PRS8065Raguel"/> and a lion indicated with <foreign xml:lang="gez">አንበሳ፡</foreign>.</desc>
+                            </decoNote>
+                            <decoNote xml:id="d3" type="miniature">
+                                <desc>Coloured picture representing the <term key="TrueCross">Cross</term> indicated with 
+                                    <foreign xml:lang="gez">መስቀለ፡ ኢየሱስ፡</foreign>, <term key="demon">Satan</term> indicated with 
+                                    <foreign xml:lang="gez">ሰይጣን፡ ጽዋሪ፡</foreign>, the Virgin Mary indicated with 
+                                    <foreign xml:lang="gez">ጽዮን፡</foreign>, a tabot indicated with <foreign xml:lang="gez">ታቦት፡</foreign>, 
+                                    the Saviour of the World indicated with <foreign xml:lang="gez">መድሃኔ፡ ዓለም፡</foreign> under an umbrella
+                                    indicated with <foreign xml:lang="gez">ድባብ፡</foreign>. Below the Virgin Mary indicated with 
```

Comment:
Should it be "Below, …"? Or shoud the "and" in the next line be taken away and replaced with a comma? I don't understand the syntax here, but I think something needs to be changed.